packaging/utils/coverity-scan.sh

```diff
@@ -37,10 +37,12 @@
 
 set -e
 
-INSTALL_DIR="/opt"
+if [ "$(uname -s)" != "Linux" ] || [ "$(uname -m)" != "x86_64" ]; then
+  echo "This script can only be used on a 64-bit x856 Linux system."
```

Comment:
@Ferroin we have a typo here (`x856`).

Reply:
On my local environment the script worked, I am only expecting the small change for this typo.